src/routes/Agreements/AgreementsView.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import { JCheckbox, JText, JRaisedButton } from 'components/base'
+import { setAgreements, getAgreements, checkAgreements } from 'utils/agreements'
+
+type StateProps = {|
+  +isDisabled: boolean,
+|}
+
+const conditionsArray = [
+  `I understand that my funds are stored securely on my personal
+  computer. No private data is sent to Jibrel AG servers.
+  All encryption is done locally in browser`,
+  `I consent that Jwallet service is provided as is without warranty.
+  Jibrel AG does not have access to my private information and could not
+  participate in resolution of issues concerning money loss of any kind`,
+  `I consent to allow cookies for collecting anonymous usage data to improve
+  quality of provided service`,
+]
+
+const nameConditionsArray = ['condition-0', 'condition-1', 'condition-2', 'condition-3']
+
+class AgreementsView extends PureComponent<Props, StateProps> {
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      isDisabled: true,
+    }
+  }
+
+  onChange = (nameConditions: NameConditionsType) => () => {
+    setAgreements(nameConditions)
+    this.setState({ isDisabled: !checkAgreements(nameConditionsArray) })
+  }
+
+  render() {
+    const {
+      isDisabled,
+    }: StateProps = this.state
+
+    const agreementsFlags = getAgreements()
+    return (
+      <div className='agreements-view'>
+        <div className='content'>
+          <h1 className='title'>
+            <JText
+              size='title'
+              color='white'
+              value='Terms and Conditions'
+              whiteSpace='wrap'
+              align='center'
+            />
+          </h1>
+          <div className='items'>
+            {conditionsArray.map((item: string, i) => (
```

Comment:
need to use some ids, by condition name I asume
and avoid using of array indexes as key attr